bdai_ros2_wrappers/test/test_wait_for_messages.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
```

Comment:
@kaiyu-zheng nit^2: could we relocate this to `test_subscription.py` for consistency?

Reply:
OK, I can do that. I thought about it at first but was hesitant because there are two node classes created specifically for this test, not for other tests in `test_subscription.py`. What's your take on this concern?

Reply:
Done.